backoffice/src/Data/DatabaseSeeder.cs

```diff
@@ -169,7 +169,9 @@ private static Droit SeedDroit(Wreck wreck, Salvor salvor, ApplicationUser user)
             GoodsDischargedBy = Faker.Name.FullName(),
             DateDelivered = Faker.Date.Between(reportedDate, DateTime.UtcNow).ToShortDateString(),
             Agent = Faker.Name.FullName(),
-            RecoveredFrom = Faker.Random.ArrayElement(new[] { "Afloat", "Ashore", "Seabed" }),
+            RecoveredFrom = Enum.GetValues(typeof(RecoveredFrom))
```

Comment:
This was the legacy one, leave it as it was for legacy, and do this earlier on with the location stuff